### PR TITLE
ENG-4652 update code valid

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

File: src/helpers/attrValidation.js [hunk 1]
```diff
@@ -215,3 +215,12 @@ export const imageValidate = memoize(langCode => (input) => {
...
 }, (...args) => JSON.stringify(args));
+
+export const codeWithDash = value =>
+  (value && /^[0-9a-zA-Z_.-]+$/i.test(value) ? undefined : (
+    <FormattedMessage id="validateForm.codeWithDash" />
+  ));
+export const friendlyCodeWithDash = value =>
+  (value && /^[0-9a-zA-Z_.-]+$/i.test(value) ? undefined : (
+    <FormattedMessage id="validateForm.friendlyCodeWithDash" />
+  ));
```
Comment (ichalagashvili) [inline src/helpers/attrValidation.js:226]: why do we need 2 separate validators? is not just `codeWithDash` enough?

**Review by ichalagashvili — CHANGES_REQUESTED** [review]
please fix the eslint/prettier changes.

File: src/helpers/attrValidation.js (hunk 1 above)
Comment (DMinnai) [reply src/helpers/attrValidation.js:226]: I did include 2 different validator because intl messages included are different.

**Review by ichalagashvili — APPROVED** [review]

File: src/helpers/attrValidation.js (hunk 1 above)
Comment (DMinnai) [reply src/helpers/attrValidation.js:226]: I did reopen the pull for the necessary test

**Pull request merged**